packages/doc-ai-react/src/App.tsx

```
@@ -1,12 +1,34 @@
 import PriceProtectionPage from './components/PriceProtectionPage'
+import { createMcpServer } from './mcp-servers'
 import './App.css'
 
+// 路由管理器组件 - 在 Router 上下文中设置导航器
+function RouterManager() {
+  const navigate = useNavigate()
+
+  useEffect(() => {
+    // 设置导航器
+    setNavigator(async (route) => {
+      debugger
```

Biome (2.4.6)

[error] This is an unexpected use of the debugger statement.

(lint/suspicious/noDebugger)